### ISSUE TITLE
Remove redundant wrapper contexts

### ISSUE THREAD

**Issue by arwhyte** [IMSGlobal/caliper-contexts#98]
I propose that we remove redundant "wrapper" JSON-LD context documents that do no more than wrap the mother ship context, as in:

```
{
  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1"
}
```
There is no real gain doing this and introduces unnecessary choice into the JSON-LD context equation.  

This thinning will leave in place `-extension` contexts.

**Comment by ViktorHaag** [comment]
I like this proposal -- it makes it cleaner and easier to see which contexts people should be using in their events (i.e. either the base context, or an extension context).

**Comment by arwhyte** [comment]
Resolved by #101
